bebop_driver/src/bebop_driver_nodelet.cpp

```diff
@@ -392,6 +412,19 @@ void BebopDriverNodelet::TakeSnapshotCallback(const std_msgs::EmptyConstPtr &emp
   }
 }
 
+void BebopDriverNodelet::SetPictureFormatCallback(const std_msgs::UInt8ConstPtr& format_ptr)
+{
+  try
+  {
+    ROS_INFO("Setting picture format to %f", format_ptr->data);
+    bebop_ptr_->SetPictureFormat(format_ptr->data);
```

Comment:
What if `format_ptr->data` is not set to a value supported by `SetPictureFormat`? Please add a guard (or a warning) against this.